### DISCUSSION TITLE
채팅방

### DISCUSSION THREAD

**Discussion by kikiru328** [kikiru328/Bone_Detection#26] [CHAT]
떠드는 곳
![image](https://user-images.githubusercontent.com/60537388/137920346-39967791-c9c8-4d21-b842-b1232a79dd4a.png)

**Comment by kikiru328** [comment]
opencv 공부하기 어렵네여

**Reply by hye-jj** [reply]
넘파이의 부분 배열 빼내기 << 이게 어떤 부분인지 잘 모르겠어요ㅠㅠㅠ

**Reply by SHEYCHAE** [reply]
졸려요

**Reply by SHEYCHAE** [reply]
> 넘파이의 부분 배열 빼내기 << 이게 어떤 부분인지 잘 모르겠어요ㅠㅠㅠ

이게 imagedatagenerator 사용해서 넘파이 배열 추출 하는 부분인거같아요~

**Reply by hye-jj** [reply]
> > 넘파이의 부분 배열 빼내기 << 이게 어떤 부분인지 잘 모르겠어요ㅠㅠㅠ
> 
> 이게 imagedatagenerator 사용해서 넘파이 배열 추출 하는 부분인거같아요~

아하ㅠㅠㅠㅠ 감사합니다!

**Reply by SHEYCHAE** [reply]
# memo 
imagegenerator conv_base 모댈 predict method 이미지 특성 추출

**Reply by kikiru328** [reply]
활발하네요

**Reply by SHEYCHAE** [reply]
/content/drive/MyDrive/cakd_colab/2차플젝/데이터 원본/BA_total.pkl
여기에 df_toal 피클로 올려놨음 ㅎㅎ

혹~~~시  모를 수도 있으니까 colab에서 피클 로드하는법
import pickle5 as pickle
with open("/content/drive/MyDrive/cakd_colab/2차플젝/데이터 원본/BA_total.pkl",'rb') as f:
  df = pickle.load(f)

**Comment by SHEYCHAE** [comment]
궁금한거1. 욜로를 학습시키면 결과가 어떻게 나와요? 
 2. 사진 넣으면 알아서 바운딩박스 보이게하는?건 어케해요?
3. TJnet으로 어케연결? 학습시킨 모델을 Tj로 또 연결하는건가??



1- A : 혜정이언니 깃허브에 yolo학습 올라와 있고 하는법 설명받았으니 기억하기(한번 해보기? )

2-A : 1번코드에서 확인 가능하고 , 모델에 사진 넣어주면 알아서 크롭해주고 크롭한거 저장도됨, 그리고 욜로 각각 roi 바운딩박스 쳐져서 나온다.

3-A : 종현이오빠 깃허브에 Tjnet 코드 올라와있고, dataframe에 각 roi별 파일 이름 넣은 df만들어서 돌리면 됨
